python/gilded_rose.py

```diff
@@ -6,34 +6,39 @@ def __init__(self, items):
         self.items = items
 
     def update_quality(self):
+
         for item in self.items:
-            if item.name != "Aged Brie" and item.name != "Backstage passes to a TAFKAL80ETC concert":
-                if item.quality > 0:
-                    if item.name != "Sulfuras, Hand of Ragnaros":
-                        item.quality = item.quality - 1
-            else:
-                if item.quality < 50:
-                    item.quality = item.quality + 1
-                    if item.name == "Backstage passes to a TAFKAL80ETC concert":
-                        if item.sell_in < 11:
-                            if item.quality < 50:
-                                item.quality = item.quality + 1
-                        if item.sell_in < 6:
-                            if item.quality < 50:
-                                item.quality = item.quality + 1
             if item.name != "Sulfuras, Hand of Ragnaros":
                 item.sell_in = item.sell_in - 1
-            if item.sell_in < 0:
-                if item.name != "Aged Brie":
-                    if item.name != "Backstage passes to a TAFKAL80ETC concert":
-                        if item.quality > 0:
-                            if item.name != "Sulfuras, Hand of Ragnaros":
-                                item.quality = item.quality - 1
+
+            if item.name == "Aged Brie":
+                if item.quality < 50:
+                    item.quality = item.quality + 1
+
+            elif item.name == "Sulfuras, Hand of Ragnaros":
+                item.quality = item.quality
+
+            elif item.name.split(" ")[0] == "Backstage" and item.name.split(" ")[1] == "passes":
```

Comment:
You can use [startswith](https://www.w3schools.com/python/ref_string_startswith.asp) here 😄